Assembly/Cleanup.bat

```diff
@@ -0,0 +1,4 @@
+cd ..
+del /S patchData.json
+del /S exported_symbols.txt
+del /S enemizerBasePatch.json
```

Comment:
This file matches the `Assembly` project pre-build. Is this file necessary?

Reply:
There is a call to this batch file.
In assembly.csproj, towards the end, are these lines.
    <Target Name="BeforeClean">
        <Exec Command="Cleanup.bat" ContinueOnError="true" />
    </Target>

This allows the files to be cleaned up, when you call clean project.